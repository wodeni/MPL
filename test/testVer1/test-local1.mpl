int foo()
{
    int i; 
/* Should hide the formal i */

    i = 42; 
    print(i + i);
    return 42;
}

int main()
{
    Mat<int>[3][3] m;
    int i;
    i = 8;
    print(i);
    m = [1,1,1;1,1,1;1,1,1];
    printm(m);
    foo @ m;
    print(i);
    printm(m);
    return 0;
}
