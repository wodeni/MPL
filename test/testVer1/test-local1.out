8
[1, 1, 1; 1, 1, 1; 1, 1, 1; ]
84
84
84
84
84
84
84
84
84
8
[42, 42, 42; 42, 42, 42; 42, 42, 42; ]
